source/local/local.go

```diff
@@ -37,6 +38,13 @@ const Name = "local"
 // LabelFeature of this feature source
 const LabelFeature = "label"
 
+// ExpiryDateKey is the key of this feature source indicating
+// when features should be removed.
+const ExpiryDateKey = "# +expiry-time"
```

Comment:
If we want to define a const, align the name

```suggestion
const ExpiryTimeKey = "# +expiry-time"
```